wsgi/odyssy/people/models.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import unicode_literals
+from django.core.urlresolvers import reverse
+
+from django.db import models
+
+
+class People(models.Model):
+    class Meta:
+        ordering = ('name',)
+        verbose_name = 'People'
+        verbose_name_plural = 'People'
+
+    STATUS_CHOICES = (
+        ('faculty', 'Faculty'),
+        ('staff', 'Staff'),
+        ('visiting_faculty', 'Visiting Faculty'),
+    )
```

Comment:
Them how you categorize them in which choice. They are not faculty (`Registrar`), `Director` is not just come under staff, It is a post of administrative head. Shouldn't we think again over `CHOICES`